back/spec/services/local_project_copy_service_spec.rb

```diff
@@ -132,6 +132,20 @@
         .to eq source_project.causes.first.to_json(except: %i[id participation_context_id image updated_at created_at])
     end
 
+    it 'copies associated custom_forms & related custom_fields & custom_field_options' do
+      custom_form = create(:custom_form)
+      custom_field = create(:custom_field, resource_type: 'CustomForm', input_type: 'select', resource_id: custom_form.id)
+      create(:custom_field_option, custom_field_id: custom_field.id)
+      source_project = Project.find(custom_form.participation_context_id)
```

Comment:
You can just use `custom_form.participation_context` without any additional variables.